### DISCUSSION TITLE
LTAR filter [Sep 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3884] [Discord Threads]
> 🙋‍♂️ **emdax**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Use filter? You intend to do it over API?  
![image0](https://cdn.discordapp.com/attachments/1019856841552965672/1019858591848595496/Screenshot_2022-09-15_at_12.01.14_PM.png)  
#  
> 🙋‍♂️ **emdax**  
#### Yes API  
#  
> 💁 **o1lab**  
#### pls read the autogenerated swagger api for your project.    
####    
#### apis & support > swagger > nested[table][where] will be query param   
####    
#### Although, we can help - most of it boils down to reading docs 😅  
#  
> 🙋‍♂️ **emdax**  
#### thanks!  
#  
> 🙋‍♂️ **emdax**  
#### in docs it says:  
#  
> 🙋‍♂️ **emdax**  
#### nested[_nc_m2m_ccaa92x__q List][where]=(field1,eq,value)  
#  
> 🙋‍♂️ **emdax**  
#### what is _nc_m2m_ccaa92x__q ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1019856841552965672)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
